### ISSUE TITLE
Weaviate adapter ignore table name and created class "Content"

### ISSUE THREAD

**Issue by hsm207** [dlt-hub/dlt#1644]
### dlt version

 0.5.1

### Describe the problem

When ingesting a table named "products" in weaviate, the corresponding class in weaviate is "Content".

### Expected behavior

The corresponding class in weaviate should be "Products" because I named the table "Products"

### Steps to reproduce

1. create a postgreql instance with table name products and columns product & description
2. create a weaviate instance
3. Run this script:
```
from sql_database import sql_database
from dlt.destinations.adapters import weaviate_adapter
import dlt

products_table = sql_database().with_resources("products")

pipeline = dlt.pipeline(
        pipeline_name="postgres_to_weaviate_pipeline",
        destination="weaviate",
    )

products_table.table_name = "Products"

info = pipeline.run(
    weaviate_adapter(products_table, vectorize="description"),
)

print(info)
```

### Operating system

Linux

### Runtime environment

Local

### Python version

3.11

### dlt data source

sql_database

### dlt destination

Weaviate

**Comment by rudolfix** [comment]
@sh-rp is your PR #1645 fixing the above?

@hsm207 the problem is that you are adapting a whole source, not a single resource. to some degree you are right if you select a single resource :) let me comment on PR above. a quick fix:
```
from sql_database import sql_database
from dlt.destinations.adapters import weaviate_adapter
import dlt

products_table = sql_database().with_resources("products")

pipeline = dlt.pipeline(
        pipeline_name="postgres_to_weaviate_pipeline",
        destination="weaviate",
    )

products_table.table_name = "Products"

info = pipeline.run(
    weaviate_adapter(products_table.products, vectorize="description"),
)

print(info)
```

**Comment by rudolfix** [comment]
fixed and will be released in `0.5.2`
